addons/hr_homeworking/static/src/calendar/common/calendar_common_renderer.js

```diff
@@ -16,6 +16,19 @@ patch(AttendeeCalendarCommonRenderer.prototype, "hr_homeworking_calendar_common_
                 return "<div>" + moment(date).format("dddd") + "</div>";
               }
         });
+        return Object.assign(a, {
+            eventOrder: function(event1, event2){
+                if (event1.id.includes("A")){
+                    return -1;
+                } else {
+                    if(event2.id.includes("A")){
+                        return 1;
+                    } else {
+                        return event1.title.localeCompare(event2.title);
+                    }
+                }
+            }
+        })
```

Comment:
Note that you can assign more than one property to the object, so you can just do

```js
return Object.assign(this._super(), {
    columnHeaderHtml: function () {...},
    eventOrder: function() {...},
})
```